src/_posts/2019-12-31-test-driving-a-cdn.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+asset-type: post
+name: test-driving-a-cdn
+title: Test Driving a CDN
+date: 2019-12-31 10:00:00 +00:00
+author: Christopher Eyre
+image:
+    src: /assets/custom/img/blog/swan.png
+tags:
+    - varnish
+    - CDN
+    - TDD
+abstract: How to test drive a CDN
+alias: [/2019/12/31/test-driving-a-cdn]
+---
+
+# How to test drive a Content Delivery Network
+
+## The problem being solved
+
+How not to break the internet:  
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/T73h5bmD8Dc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Introduction
```

Comment:
The introduction section is quite long, it may be worth adding more sections. I feel like line 54 "The sample code..." is where the introduction about CDNs has really ended; perhaps a `## How` heading here?